src/codegen.cpp

```diff
@@ -1942,7 +1943,8 @@ class jl_codectx_t {
       : builder(llvmctx),
         emission_context(params),
         call_targets(),
-        world(params.world),
+        min_world(params.min_world),
+        max_world(params.max_world),
```

Comment:
These won't usually be correct values, since the min_world/max_world that is supposed to be referred to here is that of the CodeInstance object which held the IR that is being compiled, not the content the params struct.